src/dialogs/stat_dialogs/chart_options_view.pas

```diff
@@ -0,0 +1,286 @@
+unit chart_options_view;
```

Comment:
Something is wrong in the creation of the chart option view. I do not see the chart min/max value input anywhere!
![image](https://user-images.githubusercontent.com/16835936/225700435-96dcdb4a-61f7-4347-87e2-9049d7e855e5.png)

Reply:
Two things:
1. you must select variables first (depending on the graph type). Once the variables are selected, the dialog will know if dates are possible.
2. x and y min/max don't show for every graph type; only where it makes sense
<img width="583" alt="Screen Shot 2023-03-16 at 14 39 10" src="https://user-images.githubusercontent.com/855040/225721242-20cd2cd5-7c02-4698-8aff-f862315ba4b7.png">
<img width="300" alt="Screen Shot 2023-03-16 at 14 39 16" src="https://user-images.githubusercontent.com/855040/225721273-d9983c3c-10f8-49c7-9fbd-f80085f2cce6.png">

Reply:
Something is wrong with the spacing of the input fields. Be sure the achors are correct, such that the top of the bottom field is bound to the bottom of the upper field.
![image](https://user-images.githubusercontent.com/16835936/227013093-6910baf7-c92c-44eb-8d80-c02005d1ce65.png)

Reply:
See below